go.sum

```diff
@@ -1,11 +1,12 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
-cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Did you manually remove this (and other) lines from go.sum, or was it done automatically somehow?

Reply:
Just `go mod tidy`.

Reply:
Yes, go mod tidy

Reply:
Strange, I thought `go.sum` was append-only.

Reply:
The checksum database is append only, and go.sum used to be append only, but see https://github.com/golang/go/issues/26381 for when the thought process changed and https://golang.org/ref/mod#go for where it is documented that `go mod tidy` removes unnecessary entries.